server/api/api/endpoints/model_endpoints.py

```diff
@@ -441,16 +446,18 @@ async def _get_metrics_values_params(
             "Provided only one of start time, end time. Please provide both or neither."
         )
 
-    metrics_and_results = [
-        mm_endpoints._parse_metric_fqn_to_monitoring_metric(fqn) for fqn in name
-    ]
     metrics = []
     results = []
-    for m in metrics_and_results:
-        if m.type == mm_endpoints.ModelEndpointMonitoringMetricType.METRIC:
-            metrics.append(m)
+    for fqn in name:
+        metric = mm_endpoints._parse_metric_fqn_to_monitoring_metric(fqn)
+        if metric.project != project:
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                f"Metric '{fqn}' does not belong to the project '{project}'."
```

Comment:
Might be nice to say which project it does belong to, since we have this information.